### PR TITLE
build on top of cdh5-base for hadoop setup

### PR DESCRIPTION
**Pull request by jincheng-factual**
@CMBleakley

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
-FROM boritzio/docker-base
+FROM factual/docker-cdh5-base
```
Comment (CMBleakley) [inline Dockerfile]: can you set the tag to ```java8```

Comment (jincheng-factual) [inline Dockerfile]: done

**Comment by CMBleakley** [comment]
OK, I merged the changes to branch 1.1.0 (also tagged 1.1.0 in dockerhub)

**Comment by CMBleakley** [comment]
images are building now: https://hub.docker.com/r/cmbleakley/docker-postgis-with-bootstrap/builds/

**Pull request closed without merging**